client/components/jetpack-header/woocommerce.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Module variables
+ */
+const lightColorSchemeLogo = (
+	<g fillRule="nonzero" transform="translate(360)">
+		<path
+			d="M23.7,0.2 L222.8,0.2 C235.4,0.2 245.6,10.4 245.6,23 L245.6,99 C245.6,111.6 235.4,121.8 222.8,121.8 L151.4,121.8 L161.2,145.8 L118.1,121.8 L23.8,121.8 C11.2,121.8 1,111.6 1,99 L1,23 C0.9,10.5 11.1,0.2 23.7,0.2 Z"
+			id="Shape"
+			fill="#342148"
```

Comment:
Is this fill overwritten with CSS later and how does it play with the dark color theme in Calypso?

Reply:
`lightColorSchemeLogo` doesn't actually get used. `darkColorSchemeLogo` does, and I do not overwrite the colors with CSS.

I just wanted to provide a version for completeness as all of the other co-branded logos have alternatives.

Reply:
Sounds good!